.github/workflows/regressions.yml

```diff
@@ -0,0 +1,72 @@
+name: Regressions
+on: workflow_dispatch
+
+env:
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 1
+  PUPPETEER_EXECUTABLE_PATH: /usr/bin/google-chrome
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    name: Build W3C profile
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-12-${{ hashFiles('**/package-lock.json') }}
+      - run: npm ci
+      - run: npm run build:w3c
+      - uses: actions/upload-artifact@v2
+        with:
+          name: builds
+          path: |
+            builds/respec-w3c.js*
+            builds/respec-worker.js*
+            builds/respec-highlight.js*
+          retention-days: 3
+
+  test:
+    name: Diff ${{ matrix.source }}
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        source:
+          - https://w3c.github.io/manifest/
+          - https://w3c.github.io/payment-request/
+          - https://w3c.github.io/trace-context/
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-12-${{ hashFiles('**/package-lock.json') }}
+      - run: npm ci --production --ignore-scripts
+      - uses: actions/download-artifact@v2
+        with:
+          name: builds
+          path: ./builds/
+      - run: git status
+      - run: node ./tools/respec2html ${{ matrix.source }} before.html -t 30 --verbose
+      - run: node ./tools/respec2html ${{ matrix.source }} after.html -t 30 --verbose --use-local
+      - name: Install git-delta # prettier git diff
+        run: |
+          set -v
+          wget -q -O delta.tar.gz https://github.com/dandavison/delta/releases/download/0.7.1/delta-0.7.1-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
wonder if this versioned software will become a problem to maintain? Probably not... just a thought.

Reply:
I was thinking of converting it into a separate GitHub action, which I can update as needed.
`cargo install` took ~4min, so had to use a pre-compiled binary.

Reply:
Ok, but time taken on Ci shouldn't be an issue right? That's the "cost of doing business".

Reply:
No issues now as I've moved this into a separate GitHub action. It takes 1s to install rather than 4min.

![image](https://user-images.githubusercontent.com/8426945/114393597-cfa5c480-9bb7-11eb-9862-47d69b7f56c0.png)

Reply:
Your meme game is strong, Padawan.